dexcom/fetch/runner.go

```diff
@@ -538,6 +532,23 @@ func (t *TaskRunner) fetchData(startTime time.Time, endTime time.Time) (data.Dat
 	return datumArray, nil
 }
 
+func (t *TaskRunner) fetchDataRangeStart() (time.Time, error) {
+	t.logger.Debug("fetching dataRange ...")
+	response, err := t.DexcomClient().GetDataRange(t.context, t.tokenSource)
```

Comment:
Contexts, by their nature, are specific to a given function call, and should be passed as the first argument to the method. https://go.dev/wiki/CodeReviewComments#contexts

Reply:
Looks like this is just how TaskRunner is setup though, so I guess disregard. It goes against the usual convention, but as long as our tasks aren't multithreaded, it should be fine I guess.